.drone.star

```diff
@@ -368,6 +368,18 @@ def buildOcisBinaryForTesting(ctx):
   }
 
 def uploadCoverage(ctx):
+  sonar_env = {
+      'SONAR_TOKEN': {
+        'from_secret': 'sonar_token',
+      },
+    }
+  if True or ctx.build.event == 'pull_request':
```

Comment:
Why is `True` here?
The code in the `if` will always execute.

Reply:
This was a leftover from testing and I removed it.

But somehow I am now confused if it is `ctx.build.event == 'pull'` or `ctx.build.event == 'pull_request'`!? Docs (https://docs.drone.io/pipeline/triggers/#by-event) only mention `pull_request` but `pull` seems to somehow work in our current pipeline.

Reply:
Do both events work? I see a lot of repos where we are using both at the same time.

Reply:
I see no way that both work, as a string can not equal two different strings...

`pull_request` is the right one. I had a look at our other repos: https://github.com/search?p=1&q=org%3Aowncloud+%22ctx.build.event%22+filename%3A.drone.star&type=Code